episodes/DC-explanation.md

```diff
@@ -0,0 +1,98 @@
+---
+title: "Explain the Decision Tree"
+teaching: 10
+exercises: 1
+---
+
+## Why did we introduce a decision tree at the beginning of our questionnaire?
+
+Part of the challenge in filling out a Software Management Plan (SMP) is to understand and define which points in the SMP are important, or even relevant, for your software. Depending on its scope and the impact you foresee it having, your software might need more attention in some areas of Software Management and less in others. We have decided to introduce a Decision Tree in our online questionnaire to help you define what is relevant in the Plan by answering at most four simple questions. And so, after introducing the name of your project and its purpose, you will enter the Decision Tree part of the tool. The figure below shows how the decision tree works:
+
+Figure
+![Decision Tree Layout](../img/Decision_Tree_graphic.svg)
+
+The reasoning behind the structure of the decision tree is to allow you to jump out of the process and into filling out the SMP as soon as possible. As you might notice from the figure above, the more people are involved in a project, the more complex the SMP is going to be.
```

Comment:
That you go into filling in the SMP instead of continuing with the Decision Tree. Changing it to: "is to allow you to start filling in the SMP, as soon as all relevant topics for your software are identified. You don't need to navigate the entire decision tree unless all topics in the SMP are relevant to you."